media/inf/input.inf

```diff
@@ -67,24 +65,22 @@ hid.dll
 AddService = hidusb, 0x00000002, HID_Service_Inst
 
 [HID_Service_Inst]
-ServiceType   = 1
-StartType     = 3
-ErrorControl  = 1
+ServiceType = 1
+StartType = 3
+ErrorControl = 1
 ServiceBinary = %12%\hidusb.sys
 LoadOrderGroup = Extended Base
 
 [Strings]
 ; Non-localizable
-ReactOS = "ReactOS Project"
+ReactOS="ReactOS Project"
```

Comment:
Please keep at least one space around the equals sign. It makes otherwise hard to read where the key name ends and the value starts. (Comment valid for all the touched files.)

Reply:
Many files and key-value-pairs do not follow that rule already. I harmonized the strings section within that file to the most compact style here, which is also used throughout hdc.inf, and I did it in input.inf because I had to touch each line here anyway, see I even did that in two steps, read my 2 commit comments each:
https://github.com/reactos/reactos/pull/4952/commits/6b4baed215aeb4c13db0f2b05edd46edd1b80411 (harmonizing most to the section that was in superior state already), and then concluded:
https://github.com/reactos/reactos/pull/4952/commits/6b8fb71ac3e0562f54b4d91d36eec3997a582c39